### PR TITLE
III-4815 Add category uitpas to enum of priceInfo

### PR DESCRIPTION
**Pull request by JonasVHG**
### Changed

- Changed `Enum` of `PriceInfo` to allow `uitpas` as a category

---

Ticket: https://jira.uitdatabank.be/browse/III-4815

### CODE REVIEW

File: projects/uitdatabank/models/common-priceInfo.json [hunk 1]
```diff
@@ -21,10 +58,11 @@
             "type": "string",
             "enum": [
               "base",
-              "tariff"
+              "tariff",
+              "uitpas"
             ],
             "example": "base",
-            "description": "Indicates whether this is the base price or a custom tariff.\n`base` for the base price, and `tariff` for custom/extra tariffs."
+            "description": "Indicates whether this is the base price or a custom tariff.\n`base` for the base price, and `tariff` for custom/extra tariffs. For compability `uitpas` is allowed but ignored."
```
Comment (bertramakers) [inline projects/uitdatabank/models/common-priceInfo.json:65]:
```suggestion
            "description": "Indicates whether this is the base price or a custom tariff.\n`base` for the base price, and `tariff` for custom/extra tariffs. Tariffs with category `uitpas` are read-only and are updated automatically recalculated when other tariffs change."
```

File: projects/uitdatabank/models/common-priceInfo.json [hunk 2]
```diff
@@ -42,8 +80,7 @@
             "description": "The currency of the price. Currently only `EUR` is supported."
           },
           "name": {
-            "$ref": "./common-name.json",
-            "description": "Name of a tariff inside priceInfo. Required if category is set to `tariff`. For the `base` category UiTdatabank will set a default. Requires at least one value, for the language specified in the `mainLanguage` property. Names must be unique (per language) when using multiple tarrifs."
```
Comment (bertramakers) [inline projects/uitdatabank/models/common-priceInfo.json:46]: Why was this description removed?

File: projects/uitdatabank/models/common-priceInfo.json [hunk 3]
```diff
@@ -12,6 +12,43 @@
...
+        "category": "tariff",
+        "price": 8,
+        "priceCurrency": "EUR",
+        "name": {
+          "nl": "Jongeren (<18j)",
+          "en": "Youth (<18yo)"
+        }
+      }
+    ]
+  ],
```
Comment (bertramakers) [inline projects/uitdatabank/models/common-priceInfo.json:51]: Please discard changes like these in git whenever Stoplight re-shuffles properties, as it results in a confusing diff

**Review by LucWollants — APPROVED** [review]
Thanks for fixing the Stoplight Shuffle ™ 😀

**Review by bertramakers — APPROVED** [review]

**Comment by bertramakers** [comment]
Stoplight Shuffle ™ 🕺

**Pull request merged**